### ISSUE TITLE
Restyle Map Embed to work in a smaller context

### ISSUE THREAD

**Issue by caharding** [ushahidi/platform-pattern-library#53]
Suggestion 
![map embed](https://cloud.githubusercontent.com/assets/1618793/15082172/298fc0f6-137b-11e6-8e59-b27ac4049ec7.png)

**Comment by caharding** [comment]
Suggest restyling the timeline and post in a similar manner

**Comment by Firenz** [comment]
Hi! I'm Alicia, an Outreachy candidate looking for some issues to do in this repository.

@Angamanga I just found this issue opened, but it seems like it has already been fixed?

I just opened the embed map, checked in a small resolution, and it already works fine in a small context.

I attach two screenshots to showcase this:
![2020-10-15_18-57-13](https://user-images.githubusercontent.com/2558299/96162145-7a466e80-0f18-11eb-96ec-20ce5b0c5b7b.png)
![2020-10-15_18-57-39](https://user-images.githubusercontent.com/2558299/96162159-7e728c00-0f18-11eb-881e-ee45af3cfd24.png)
